nuxt.config.js

```diff
@@ -205,7 +205,8 @@ module.exports = {
     '~/plugins/page',
     '~/plugins/vue-filters',
     '~/plugins/vue-directives',
-    { src: '~/plugins/vue-announcer', mode: 'client' }
+    { src: '~/plugins/vue-announcer', mode: 'client' },
+    { src: '~/plugins/vue/vue-masonry', ssr: false }
```

Comment:
We are now using three different syntaxes in the plugin definitions for specifying if a plugin should be loaded only on client side. Let's choose one!
```javascript
'~/plugins/hotjar.client'
{ src: '~/plugins/vue-announcer', mode: 'client' },
{ src: '~/plugins/vue/vue-masonry', ssr: false }
```
See https://nuxtjs.org/docs/2.x/directory-structure/plugins/#name-conventional-plugin
and
https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-plugins/

Reply:
Good point, thanks!